### ISSUE TITLE
[Plugin SubtitleSwitcher] Doesn't have an option to select accessibility subtitle

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#10189]
@DanielnetoDotCom Danny, I noticed that the Subtitle plugin doesn't have an option to select whether the subtitle is an accessibility subtitle, such as Descriptive Captions (CC), used by people with hearing impairments.

It's important to differentiate between a normal subtitle and a descriptive subtitle.

I think there could be a selector that indicates whether the subtitle is an accessibility subtitle or a descriptive subtitle.

It's important to have this option for inclusion and accessibility, something that is essential in audiovisual media today.

<img width="521" height="407" alt="Image" src="https://github.com/user-attachments/assets/0746c15e-7f96-4492-b7c9-7a48e5d0e4e6" />

<img width="308" height="220" alt="Image" src="https://github.com/user-attachments/assets/22ad23d0-7cf9-43c6-8e33-5bcf68108462" />

Examples of descriptive captions:

<img width="768" height="244" alt="Image" src="https://github.com/user-attachments/assets/28b85ba4-f7b3-4162-b586-0c29be1e2b30" />

https://player.vimeo.com/video/790464662
